source/game_sa/Tasks/TaskTypes/TaskSimpleClimb.h

```diff
@@ -6,10 +6,12 @@
 */
 #pragma once
 
-
 #include "TaskSimple.h"
 #include "AnimBlendAssociation.h"
 #include "Entity.h"
+#include <extensions/EntityRef.hpp>
+#include <eSurfaceType.h>
+
 
```

Comment:
remove one extra line, gh doesn't show the diff properly for some reason
```suggestion
#include "TaskSimple.h"
#include "AnimBlendAssociation.h"
#include "Entity.h"
#include <extensions/EntityRef.hpp>
#include <eSurfaceType.h>

```

Reply:
we usually leave 2 lines between the includes and the code, no?